docs/reference/requirement-sets/ribbon-api-requirement-sets.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Ribbon API requirement sets
+description: 'Specifies which Office platforms and builds support the dynamic ribbon APIs.'
+ms.date: 05/28/2020
+ms.prod: non-product-specific
+localization_priority: Normal
+---
+
+# Ribbon API requirement sets
+
+Requirement sets are named groups of API members. Office Add-ins use requirement sets specified in the manifest or use a runtime check to determine whether an Office host supports APIs that an add-in needs. For more information, see [Office versions and requirement sets](/office/dev/add-ins/develop/office-versions-and-requirement-sets).
+
+The Ribbon API set supports programmatic control of when custom Add-in Commands (that is, custom ribbon buttons and menu items) are enabled and disabled.
+
+Office Add-ins run across multiple versions of Office. The following table lists the Ribbon API requirement sets, the Office host applications that support that requirement set, and the build or version numbers for the Office application.
+
+|  Requirement set  | Office 2013 on Windows<br>(one-time purchase) | Office 2016 or later on Windows<br>(one-time purchase)   | Office on Windows\*<br>(connected to Office 365 subscription) |  Office on iPad<br>(connected to Office 365 subscription)  |  Office on Mac<br>(connected to Office 365 subscription)  | Office on the web  |  Office Online Server  |
+|:-----|-----|:-----|:-----|:-----|:-----|:-----|:-----|
+| RibbonApi 1.1  | N/A | N/A | Version 2002 (Build 12527.20264) or later | 16.38 or later | N/A | February 2020 | N/A|
+
+> **&#42;** During the preview phase, the Ribbon API is supported only on Excel and it requires Office 365 (the subscription version of Office). You should use the latest monthly version and build from the Insiders channel. You need to be an Office Insider to get this version. For more information, see [Be an Office Insider](https://products.office.com/office-insider?tab=tab-1). Please note that when a build graduates to the production semi-annual channel, support for preview features, including the Ribbon API, is turned off for that build.
```

Comment:
Maybe make a note vs. the *?

Reply:
Our pattern in the Req Set articles is to use asterisks in the tables. See for example, the IdentityApi and AddinCommands Req Set articles. So, for the sake of consistency, I'd like to keep  the asterisk here. We can revisit whether we should be using notes or asterisks someday, but that would be out-of-scope for this PR.